src/constants.ts

```diff
@@ -1 +1,3 @@
 export const APP_NAME = "strike-off-objections-web";
+export const OBJECTIONS_SESSION = "objections_session";
+export const OBJECTIONS_COMPANY_PROFILE = "objections_company_profile";
```

Comment:
we might have a few session constants so we can group them by prefixing SESSION_
also we know it's for objections so we can remove that bit 
eg. 
export const SESSION_COMPANY_PROFILE = "objections_company_profile"

Reply:
Updated.